standard/standard_10.php

```diff
@@ -1,8 +1,13 @@
 <?php
 /**
+ * @template TKey
+ * @template TValue
+ * @param array<TKey, TValue> $array
+ * @param callable(TValue, TKey): bool
+ * @return TValue|null
  * @since 8.4
  */
-function array_find(array $array, callable $callback): mixed {}
+function array_find(array $array, callable $callback) {}
```

Comment:
@JurianArie return type should stay here to match reflection data. And a failing test notifies about this
<img width="951" alt="Screenshot 2025-06-12 at 16 48 56" src="https://github.com/user-attachments/assets/b52d0df9-136e-4025-94ef-c6f996b5ee25" />
Other test failures are not related to your changes so I'll fix them myself. Could you please just return back `mixed` return type?

Reply:
The mixed type has been added back. That issue does indeed look related!